metadata-ingestion/tests/integration/snowflake/test_snowflake.py

```diff
@@ -30,6 +30,8 @@
 from tests.integration.snowflake.common import FROZEN_TIME, default_query_results
 from tests.test_helpers import mce_helpers
 
+pytestmark = pytest.mark.integration_batch_2
```

Comment:
I think batch 2 is already the slowest - let’s use batch 1 for some of these instead

Reply:
https://github.com/datahub-project/datahub/actions/runs/6405004395 It looks like the fastest as per the latest run